### DISCUSSION TITLE
Provide a way of accessing metadata dependencies

### DISCUSSION THREAD

**Discussion by laszlo-foldi-attentioncrm** [forcedotcom/salesforcedx-vscode#5374] [Ideas]
**Is your feature request related to a problem? Please describe.**
My pain point is, when developing portable solutions I'm always struggling with finding dependencies for my classes. For example, I have a base package for our company containing basic trigger framework, logging framework, rest framework, but when from this multi package project I only want to deploy a single package, I'm always struggling with properly extending the package xml related to said package.

**Describe the solution you'd like**
I'd like to have a way to show dependencies for either a full  package directory or a package xml. This feature already exists when creating packages from the UI and I can fetch dependencies for a single class using the 'Show Dependencies' button, but I'd like to have a similar feature in the VSCode extension. 

**Describe alternatives you've considered**
The only alternative solution I've found is, creating a package from the UI as it'll fetch all my dependencies for me. However, that's less usable in my context as I don't want to create actual packages just to manually copy all dependencies to a package xml file.

**Additional context**
![image](https://github.com/forcedotcom/salesforcedx-vscode/assets/47455710/06f88962-c82a-467b-b887-69ab0d8680a2)
![image](https://github.com/forcedotcom/salesforcedx-vscode/assets/47455710/e1c6eb49-2937-4174-9f17-d5cf7a0b1a67)
![image](https://github.com/forcedotcom/salesforcedx-vscode/assets/47455710/ad4fe5e8-623a-4562-8378-6f1ab1ca492e)

**Comment by daphne-sfdc** [comment]
Thanks @laszlo-foldi-attentioncrm for bringing this to our attention.  We will let you know if we implement your suggestion.
